### PR TITLE
Add  Member Abhijay Jain

### LINKED ISSUE #2317: [Brand] Refresh Nighthawk logos; Remove "GET" from all pages
**Current Behavior**
The current set of Nighthawk logos say "GETNIGHTHAWK". These are being replaced with a set that says "NIGHTHAWK".

**Desired Behavior**
1. Rename mega menu entry and project page to "Nighthawk". 
1. Remove "GET" from any of the descriptions of this project.
1. Rename brand section to "Nighthawk".
1. Replace old kit and logos with new kit and logos.

The website will still be getnighthawk.dev, but the logos have all dropped the "Get".

**Kit**
- [nighthawk-brand-kit.zip](https://github.com/layer5io/layer5/files/7450213/nighthawk-brand-kit.zip)

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### LINKED ISSUE #2517: [Community] Member Profile: Abhijay Jain
**Current Behavior**

@Abhijay007 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Abhijay007 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: Abhijay007
- Twitter: CodeInVeins
- LinkedIn: /abhijay-jain-551b01193
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by ShivamTyagi12345**
Signed-off-by: ShivamTyagi12345 <shivamt.ug19.ec@nitp.ac.in>

**Description**

This PR fixes #2517

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍 great work @ShivamTyagi12345 🎉

**Comment by Nikhil-Ladha** [comment]
Incorrect issue number. Please update.

File: src/collections/members/abhinay-jain/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Abhijay Jain
+position: Contributor
+image_path: ./abhijay-jain.jpg
+github: Abhijay007
+twitter: CodeInVeins
+linkedin: abhijay-jain-551b01193
+location: India
```
Comment (Nikhil-Ladha) [inline src/collections/members/abhinay-jain/index.mdx:8]: @Abhijay007 share your city name, if you would.

Comment (Nikhil-Ladha) [reply src/collections/members/abhinay-jain/index.mdx:8]: Then, @ShivamTyagi12345 make it `City, Country`

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The file/folder name is incorrect (both image, mdx), it should be `abhijay-jain` and not `abhinay-jain`

File: src/collections/members/abhinay-jain/index.mdx (hunk 1 above)
Comment (Abhijay007) [reply src/collections/members/abhinay-jain/index.mdx:8]: > Then, @ShivamTyagi12345 make it `City, Country`

`City : Delhi`

**Comment by Abhijay007** [comment]
> Incorrect issue number. Please update.

@ShivamTyagi12345 replace it with #2517

**Comment by ShivamTyagi12345** [comment]
> The file/folder name is incorrect (both image, mdx), it should be `abhijay-jain` and not `abhinay-jain`

ALL good!

**Comment by Nikhil-Ladha** [comment]
You need to rename instead of creating new files!

**Comment by iambami** [comment]
Hi @ShivamTyagi12345 any update on this?

**Comment by ShivamTyagi12345** [comment]
Sure. I have to delete the files that were committed twice. Will do it asap

**Comment by iambami** [comment]
No sweat.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@ShivamTyagi12345 just remove these two files from the PR then we are good to go :)
Screenshot from 2022-02-03 11-56-04

**Comment by Abhijay007** [comment]
Solved via #2592,  @ShivamTyagi12345 thanks for the PR

**Pull request closed without merging**